src/measure_theory/measure/finite_measure_weak_convergence.lean

```diff
@@ -279,13 +279,62 @@ end
 
 /-- Finite measures yield elements of the `weak_dual` of bounded continuous nonnegative
 functions via `finite_measure.test_against_nn`, i.e., integration. -/
-def to_weak_dual_bounded_continuous_nnreal (μ : finite_measure α) :
+def to_weak_dual_bcnn (μ : finite_measure α) :
   weak_dual ℝ≥0 (α →ᵇ ℝ≥0) :=
 { to_fun := λ f, μ.test_against_nn f,
   map_add' := test_against_nn_add μ,
   map_smul' := test_against_nn_smul μ,
   cont := μ.test_against_nn_lipschitz.continuous, }
 
+lemma to_weak_dual_bcnn_eval' (μ : finite_measure α) (f : α →ᵇ ℝ≥0) :
+  μ.to_weak_dual_bcnn f = μ.test_against_nn f := rfl
```

Comment:
I think you want this lemma instead, as a simp lemma:
```suggestion
@[simp] lemma coe_to_weak_dual_bcnn (μ : finite_measure α) :
  ⇑μ.to_weak_dual_bcnn = μ.test_against_nn := rfl
```

Reply:
I was not sure of the simp tag, since I don't know which side should really be the simp normal form. My current view is that  `test_against_nn` is a bit auxiliary, and once the lipschitzness and linearity are proven for it, and they are bundled in the `to_weak_dual_nn`, that might become the standard spelling. I.e., `⇑μ.to_weak_dual_bcnn f` could be the simp-normal form? But I don't have a clear opinion about which would work better.